tensorflow/python/compiler/tensorrt/test/test_utils.py

```diff
@@ -17,6 +17,27 @@
 import os
 
 from contextlib import contextmanager
+from functools import wraps
+
+from tensorflow.python.compiler.tensorrt.test import \
+  tf_trt_integration_test_base as trt_test
+
+
+def disable_tf32_testing(_class):
+  """Force Tensorflow to use FP32 instead of TensorFloat-32 computation.
```

Comment:
It this comment line correct in what the routine does? I thought it only disables testing using data type TF32, but the comment says it will repeat the test using F32 (which means the tests are run twice with F32).

Reply:
I don't understand your comment, the comment says: "use FP32 instead of TensorFloat-32" which means that instead of using TF32 in the unittest it is ran in FP32 mode.

The test is not executed multiple times, just TF32 kernels are not selected.

FP32/TF32 is an NVIDIA library runtime feature, not an official TF precision mode